clients/ui/frontend/src/app/pages/modelCatalog/components/ValidatedModelCard.tsx

```diff
@@ -0,0 +1,247 @@
+              <Flex
+                alignItems={{ default: 'alignItemsCenter' }}
+                gap={{ default: 'gapSm' }}
+                wrap="nowrap"
+              >
+                <span className="pf-v5-c-content pf-m-small">Average accuracy</span>
+                <Tooltip
+                  content="Average accuracy
+
+The weighted average of normalized scores from all benchmarks. Each benchmark is normalized to a 0-100 scale. All normalized benchmarks are then averaged together."
+                  position="top"
+                  className="pf-v6-c-tooltip"
+                >
+                  <HelpIcon style={{ cursor: 'help' }} />
+                </Tooltip>
+              </Flex>
+            </Flex>
+          </div>
+
+          {/* Hardware Configuration and Latency */}
+          <div style={{ paddingLeft: '8px', marginBottom: '16px' }}>
+            <Flex justifyContent={{ default: 'justifyContentSpaceBetween' }}>
+              <div>
+                <Content component={ContentVariants.small} style={{ color: '#6a6e73' }}>
+                  Hardware
+                </Content>
+                <Content component={ContentVariants.p} data-testid="validated-model-hardware">
+                  {metrics.hardwareCount}x{metrics.hardware}
+                </Content>
+              </div>
+              <div>
+                <Content component={ContentVariants.small} style={{ color: '#6a6e73' }}>
+                  RPS/rep.
+                </Content>
+                <Content component={ContentVariants.p} data-testid="validated-model-rps">
+                  {metrics.rpsPerReplica}
+                </Content>
+              </div>
+              <div>
+                <Content component={ContentVariants.small} style={{ color: '#6a6e73' }}>
+                  TTFT
+                </Content>
+                <Flex
+                  alignItems={{ default: 'alignItemsCenter' }}
+                  gap={{ default: 'gapSm' }}
+                  wrap="nowrap"
+                >
+                  <span data-testid="validated-model-ttft">{metrics.ttftMean} ms</span>
+                  <Tooltip
+                    content="Latency
+
+The delay (in milliseconds) between sending a request and receiving the first response.
+
+*TTFT (Time to First Token)
+The time between when a request is sent to a model and when the model begins streaming its first token in the response.
+
+*ITL (Inter-Token Latency)
+The average time between successive output tokens after the model has started generating.
+
+*E2E (End-to-End latency)
+The total time from when the request is sent until the last token is received."
+                    position="top"
+                    className="pf-v6-c-tooltip"
+                  >
+                    <HelpIcon />
+                  </Tooltip>
+                </Flex>
+              </div>
+            </Flex>
+          </div>
+
+          {/* Benchmarks */}
+          <div style={{ paddingLeft: '8px' }}>
```

Comment:
Same here with the inline styles - see what Stack gets you and if needed use spacing utility classes.